docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -1,25 +1,23 @@
 [[set-up-driver-plugin]]
-= Setting up the driver plugin
+= Setting Up the Driver Plug-in
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 1
 
 toc::[]
 
-[[hypervisor-drivers-overview]]
+[[setup-driver-plugin-overview]]
```

Comment:
good catch. There is no need so to make the overview anchor unique. 'overview' will just work fine.